main.go

```diff
@@ -458,6 +492,14 @@ func getVMName() string {
 	return string(vm)
 }
 
+func getInstanceID() (string, error) {
```

Comment:
If this is something specific to cloudRun, can we make the function name more descriptive? Like `getCloudRunInstanceID` maybe?